compiler/sigmatch.nim

```diff
@@ -568,7 +568,7 @@ proc inconsistentVarTypes(f, a: PType): bool {.inline.} =
 
 proc procParamTypeRel(c: var TCandidate, f, a: PType): TTypeRelation =
   ## For example we have:
-  ##..code-block:: nim
+  ## .. code-block:: nim
```

Comment:
still not correct, see my previous comment: try `nim doc -r --docInternal compiler/sigmatch.nim`

Reply:
Thanks! I got it.